k8s/upgrade/Cargo.toml

```diff
@@ -10,6 +10,7 @@ path = "src/lib.rs"
 
 [dependencies]
 openapi = { path = "../../dependencies/control-plane/openapi", default-features = false, features = [ "tower-trace" ] }
+rest-plugin = { path = "../../dependencies/control-plane/control-plane/plugin", default-features = false }
```

Comment:
This doesn't seem right pulling the rest-plugin here. We already have a client elsewhere as well right, why don't we use that?

Reply:
IMHO, i dont find any place where its used directly . 
One thing which i tried earlier was `UpgradeConfig::get_config().rest_client()` , but this didn't worked as UpgradeConfig is not initialized by the time i am calling this validation . 

Can you suggest anything here.

Reply:
Another point is that we can't use `RestClient` at all here since it won't do the port forwarding.
We have to get `UpgradeConfig` initialized before this call is done.

Reply:
@tiagolobocastro  PTAL , created a new client